code/drasil-gen/lib/Language/Drasil/Dump.hs

```diff
@@ -16,22 +16,25 @@ import Database.Drasil (traceTable, refbyTable)
 import Control.Lens ((^.))
 import System.Environment (lookupEnv)
 
+import Language.Drasil.Printers (PrintingInformation, printAllDebugInfo)
+import Text.PrettyPrint
+
 type Path = String
 type TargetFile = String
 
 -- | For debugging purposes, if the system has a `DEBUG_ENV` environment
 --   variable set to anything, we can dump the chunk maps in a system to the
 --   host system.
-dumpEverything :: SystemInformation -> Path -> IO ()
-dumpEverything si p = do
+dumpEverything :: SystemInformation -> PrintingInformation -> Path -> IO ()
```

Comment:
Why is this PR dependent on the one that deals with dumping debug information? These two should be separate. Especially as I've got qualms about that one, which will now delay this one!

Reply:
Rebased!